Updated the README.md file.

cl2112 opened the pull request:
The description of the repo on the README.md file was updated to include my name and to provide a better description of the purpose of the repo.

geoffbass reviewed (approved):
Looks good to me! (LGTM)

Generally I would probably like to see this split up into two PRs: one for updates to the README file (as the PR description suggests) and a separate one for the servlet config changes. No need to worry about splitting them here though. Out of curiosity, do you know how the servlet changes (which look like they were authored by someone else?) ended up in this branch? Maybe there's some git stuff we can align on for future PRs.

cl2112 commented:
I was also confused about the servlet changes but I just assumed it was from the original version of the repo that I cloned and did not realize that it was part of my PR. I think it was somehow added/combined when I removed my original commit (locally) to remove my personal email address from the commits. I had activated a setting on Github that checks and gives me a warning if my personal email is exposed in any of the commits I push to a repo ( I had forgotten about it, and I like messing with privacy and security options unfortunately ). My commits had my personal email on them because I had used it when setting the user.email field for git in the cloud shell, so I removed the previous commits using git rebase, changed my email to a generic one provided by Github and then created a new commit. Sorry about that. I will check my next PR to see if they are still affected.
